resources/metadata/uiVeri5ExecuteTests.yaml

```diff
@@ -0,0 +1,80 @@
+metadata:
+  name: uiVeri5ExecuteTests
+  description: Executes UI5 e2e tests using uiVeri5
+  longDescription: |
+    In this step the ([UIVeri5 tests](https://github.com/SAP/ui5-uiveri5)) are executed.
+
+#    The step is using `dockerExecute` step to spin up two containers in a Docker network:
+#    * a Selenium/Chrome container (`selenium/standalone-chrome`)
+#    * a NodeJS container (`node:lts-stretch`)
+#    In the Docker network, the containers can be referenced by the values provided in `dockerName` and `sidecarName`, the default values are `uiVeri5` and `selenium`.
+#   !!! note
+#        In a Kubernetes environment, the containers both need to be referenced with `localhost`.
+spec:
+  inputs:
+    params:
+      - name: installCommand
+        type: string
+        description: The command that is executed to install the uiveri5 test tool.
+        default: npm install @ui5/uiveri5 --global --quiet
+        scope:
+          - GENERAL
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        mandatory: true
+      - name: runCommand
+        type: string
+        description: "The command that is executed to start the tests."
+        default: /home/node/.npm-global/bin/uiveri5
```

Comment:
This works on native jenkins as well as jenkins on kubernetes. Calling just 'uiveri5' or '~/-npm-global/bin/uiveri5' did NOT work (with respect to the used environment variables).

Reply:
is it possible to add `--seleniumAddress='http://${config.seleniumHost}:${config.seleniumPort}/wd/hub'` here?

Reply:
This is not possible for two reasons:
1. the golang command execution framework only takes a single command without parameters here. Everything else will result in an error (also modifications that manipulate the command using `&&` or `|`).
2. the groovy placeholder for config-resolved parameters does not work in go.

Reply:
So you now can't have config values inside command execution? seems like quite a drawback.

Reply:
You _can_, when you call the step in your pipeline script, pass the parameter to the step and take care about getting it resolved by yourself before entering the step.

From the point of view where one has to adjust an incompatible configuration due to a breaking change it could be seen as a drawback, but for long-term future-proof DevOps strategies we think this is acceptable in order to profit from piper steps that can be compiled and run anywhere.